Makefile

```diff
@@ -133,12 +133,31 @@ test-containerized: run-etcd run-k8s-apiserver build-containerized dist/host-loc
 			ginkgo'
 	make stop-etcd
 
+# This does not currently work, kubernetes needs additional configuration
+# before the tests will run correctly.
 .PHONY: test-containerized-all-datastore
 test-containerized-all-datastore:
 	for datastore in "etcdv3" "kubernetes" ; do \
-		make test-containerized DATASTORE_TYPE=datastore; \
+		make test-containerized DATASTORE_TYPE=$$datastore; \
```

Comment:
double  $$?

Reply:
@caseydavenport double $$ is correct, I had missed that previously